vividus-plugin-mobile-app/src/main/java/org/vividus/selenium/mobileapp/MobileAppWebDriverManager.java

```diff
@@ -54,6 +55,14 @@ public MobileAppWebDriverManager(IWebDriverProvider webDriverProvider,
         this.javascriptActions = javascriptActions;
     }
 
+    @SuppressWarnings("unchecked")
+    public <T> T getSessionDetail(String detail)
+    {
+        Response response = getUnwrappedDriver(ExecutesMethod.class).execute("getSession");
```

Comment:
Maybe it could be good to cache session details, if they're immutable during the session

Reply:
> they're immutable during the session

I'm not sure about that

Reply:
So it is better to get them every time then